src/core/stdcpp/vector.d

```diff
@@ -114,6 +114,17 @@ extern(D):
     ///
     void append(T[] array)                                                  { insert(length, array); }
 
+    /// Performs elementwise equality check.
+    bool opEquals(this This, That)(auto ref That rhs)
```

Comment:
The complicated signature is because opEquals might or might not be callable with a const vector!T for any particular T.

Reply:
Is this definitely the best way to express this comparison?

Reply:
Not definitely.